### PR TITLE
use empty mesh while controller is loading and enable input while mes…

### PR DESCRIPTION
**Pull request by TrevorDev**
…h is loading

https://github.com/BabylonJS/Babylon.js/issues/3798

### CODE REVIEW

File: src/Cameras/VR/babylon.vrExperienceHelper.ts [hunk 1]
```diff
@@ -116,7 +116,17 @@ module BABYLON {
             this._laserPointer.rotation.x = Math.PI / 2;
             this._laserPointer.position.z = -0.5;
             this._laserPointer.isVisible = false;
-            this._laserPointer.parent = webVRController.mesh;
+
+            if(!webVRController.mesh){
+                // Create an empty mesh that is used prior to loading the high quality model
+                var preloadMesh = new Mesh("preloadContrllerMesh", scene);
```
Comment (deltakosh) [inline src/Cameras/VR/babylon.vrExperienceHelper.ts]: Typo in the name

File: src/Cameras/VR/babylon.vrExperienceHelper.ts [hunk 2]
```diff
@@ -116,7 +116,17 @@ module BABYLON {
             this._laserPointer.rotation.x = Math.PI / 2;
             this._laserPointer.position.z = -0.5;
             this._laserPointer.isVisible = false;
-            this._laserPointer.parent = webVRController.mesh;
+
+            if(!webVRController.mesh){
+                // Create an empty mesh that is used prior to loading the high quality model
+                var preloadMesh = new Mesh("preloadControllerMesh", scene);
+                var preloadPointerPose = new Mesh("POINTING_POSE", scene);
```
Comment (RaananW) [inline src/Cameras/VR/babylon.vrExperienceHelper.ts]: why is the naming different? Is it being used later, or just to emphasis usage?

File: src/Cameras/VR/babylon.vrExperienceHelper.ts [hunk 3]
```diff
@@ -138,6 +148,22 @@ module BABYLON {
...
+                root.name += " laserPointer";
+                root.getChildMeshes().forEach((c) => {
+                    makeNotPick(c);
+                });
+            }
+            makeNotPick(mesh);
+            var childMeshes = mesh.getChildMeshes();
+
+            for (var i = 0; i < childMeshes.length; i++) {
+                if (childMeshes[i].name && childMeshes[i].name.indexOf("POINTING_POSE") >= 0) {
```
Comment (RaananW) [inline src/Cameras/VR/babylon.vrExperienceHelper.ts]: Would be better to have POINTING_POSE stored somewhere as  a (public, readonly? ) static variable

File: src/Cameras/VR/babylon.vrExperienceHelper.ts (hunk 2 above)
Comment (TrevorDev) [reply src/Cameras/VR/babylon.vrExperienceHelper.ts]: This was just to emphasis usage.

File: src/Cameras/VR/babylon.vrExperienceHelper.ts (hunk 3 above)
Comment (TrevorDev) [reply src/Cameras/VR/babylon.vrExperienceHelper.ts]: Works for me, added

**Pull request merged**